app/models/vote.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
+
+  validates :user_id, presence: true
+  validates :work_id, presence: true
+  validates :value, presence: true, inclusion: { in: [1, -1], message: '%{value} is not a valid vote' }
+
+  def user
+    User.find(user_id)
+  end
```

Comment:
Hm... Not quite right... Why do we need an instance method named `user`? What does it mean to do `User.find(user_id)`? Where is `user_id` defined? When do we call this method?